src/crimson/osd/ec_backend.h

```diff
@@ -26,9 +26,9 @@ class ECBackend : public PGBackend
   _submit_transaction(std::set<pg_shard_t>&& pg_shards,
 		      const hobject_t& hoid,
 		      ceph::os::Transaction&& txn,
-		      osd_reqid_t req_id,
+		      OSDOpParams&& req,
 		      epoch_t min_epoch, epoch_t max_epoch,
-		      eversion_t ver) final;
+		      vector<pg_log_entry_t> logv) final;
```

Comment:
please avoid assuming `using namespace std` or `using std::vector`. always use fully qualified class name in header files to ensure that the header is self-contained. in short, s/vector/std::vector/.